src/assets/style/custom.css

```diff
@@ -16,4 +16,7 @@
 body {
     background-color: #f5f6fa;
     color: #343a40;
+}
+.pointer-hand{
```

Comment:
This could've been `scoped` to the component too, right @DannyFeliz ?

Reply:
yes, but this css class could be used by more than one component if necessary

Reply:
@jgilbertcastro I agree, I just don't see the immediate use of it and I'm kind of a fan of adding things as reusable when knowing a concrete use case for it aside from the current one. (Refactoring when seeing duplication coming along)

**But**, if you and @DannyFeliz feel good about it, no problem here.

Reply:
We can leave it like this for now, but please add a space between the class name and the open curly brace. @jgilbertcastro